server/src/main/java/org/elasticsearch/action/admin/indices/stats/ShardStats.java

```diff
@@ -211,6 +259,9 @@ static final class Fields {
         static final String PRIMARY = "primary";
         static final String NODE = "node";
         static final String RELOCATING_NODE = "relocating_node";
+        static final String SEARCH_IDLE = "search_idle";
+        static final String IS_SEARCH_IDLE = "is_idle";
```

Comment:
I'm not really happy with `search_idle` object. Thinking more about this, maybe we should add these two fields to `SearchStats`?